hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/DogTransformations.scala

```diff
@@ -40,137 +38,155 @@ object DogTransformations {
       stBatchLabel = rawRecord.getOptional("st_batch_label"),
```

Comment:
Based on the size of this file, I think we'll want to start splitting up the mapping functions by source instrument. I could get started here to make the precedent by moving `mapStudyStatus` to its own file (and moving its test to a corresponding separate spec). What do you all think?

Reply:
I think that would be a good way to start the process of splitting up the mapping logic. So all mapping/logic for "st_" fields would follow?

Reply:
I agree, that would be a good way of splitting it up 👍 this file is big enough already, and it wouldn't feel good to keep adding to it.

Reply:
Done